e2e/src/usecases/Assets.js

```diff
@@ -37,15 +40,15 @@ export default Assets = () => {
       balance: 'non zero',
       tokens: [
         {
-          tokenId: 'celo-alfajores:native',
+          tokenId: 'celo-mainnet:native',
           symbol: 'CELO',
           actions: ['Send', 'Add'],
           moreActions: ['Send', 'Add', 'Withdraw'],
           learnMore: true,
         },
         {
-          tokenId: 'celo-alfajores:0x048f47d358ec521a6cf384461d674750a3cb58c8',
-          symbol: 'TT',
+          tokenId: 'celo-mainnet:0x765de816845861e75a25fca122bb6898b8b1282a',
+          symbol: 'cUSD',
```

Comment:
I think this was meant to be a token with only `Send` action. Changing this to cUSD would mean it also has add / withdraw actions right? Maybe test asserts for presence of the action, and not absence of other actions. But would be good to include a non CICO token here to keep parity. It also says `learnMore: false` below but learn more should be visible for cusd.

Reply:
Switched to cMCO2 in 2f47622bbec84986eedfb5279964c5535b8db0a0.